agent/src/main/java/com/teamscale/jacoco/agent/PreMain.java

```diff
@@ -236,6 +236,17 @@ private static LoggingUtils.LoggingResources initializeFallbackLogging(String pr
 			return LoggingUtils.initializeDefaultLogging();
 		}
 		for (String optionPart : premainOptions.split(",")) {
+			if (optionPart.startsWith(AgentOptionsParser.DEBUG + "=")) {
+				String value = optionPart.split("=", 2)[1];
+				if (value.equalsIgnoreCase("false")) {
+					continue;
+				}
+				Path debugLogDirectory = null;
+				if (!value.isEmpty() && !value.equalsIgnoreCase("true")) {
```

Comment:
I would factor this `value.equalsIgnoreCase("true")` into a boolean variable to make the code better to read.